Expensify/App
# [User Settings] Implement Settings Menu Selectors

### Details
Original PR https://github.com/Expensify/Expensify.cash/pull/1630

### Fixed Issues
<Please replace GH_LINK with the link to the GitHub issue this Pull Request is fixing>
Fixes https://github.com//issues/1493

### Tests
- Click into Settings page from HomePage
- Click each menu item and confirm subroute renders

### Tested On


### Screenshots


#### Web

![Screen Shot 2021-03-09 at 9 04 51 AM](https://user-images.githubusercontent.com/12075600/110509386-e89df000-80b6-11eb-9905-1f0504eada7b.png)

#### Mobile Web

<img width="494" alt="Screen Shot 2021-03-09 at 9 07 04 AM" src="https://user-images.githubusercontent.com/12075600/110509452-f9e6fc80-80b6-11eb-9d52-2d8820aa4b03.png">

#### Desktop

<img width="1312" alt="Screen Shot 2021-03-09 at 9 06 15 AM" src="https://user-images.githubusercontent.com/12075600/110510093-b5a82c00-80b7-11eb-9bf3-53b0fcc55459.png">


#### iOS

<img width="494" alt="Screen Shot 2021-03-09 at 9 12 17 AM" src="https://user-images.githubusercontent.com/12075600/110510024-a0cb9880-80b7-11eb-812e-03839c2a3c74.png">

#### Android

<img width="464" alt="Screen Shot 2021-03-09 at 9 07 58 AM" src="https://user-images.githubusercontent.com/12075600/110509521-0a977280-80b7-11eb-82b5-b695ffde2d9c.png">
